int3face/chain.json

```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "int3face",
+  "chain_type": "cosmos",
+  "chain_id": "int3face-1",
+  "pretty_name": "Int3face",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://int3face.io/",
+  "bech32_prefix": "int3",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Got it, created the PR -> https://github.com/satoshilabs/slips/pull/1798

Reply:
It`s done, pr is merged